### ISSUE TITLE
Minor: please provide also the man page `unison.1` in the pre-built binaries tar files for linux

### ISSUE THREAD

**Issue by callegar** [bcpierce00/unison#1011]
That is going to help packaging the upstream binaries

**Comment by gdt** [comment]
Packaging systems should be building from source; CI is for CI, and only incidentally useful to others.

But fair that it would be helpful for users, that building the man page would be good for CI, and that because the manual is present, the man page should be.

**Comment by callegar** [comment]
> Packaging systems should be building from source.

I agree, but when there are issues with distro packages, being able to build packages from CI to make differential check or as quick fixes helps a lot! So thanks if this minor point can be addressed.
